moveit_core/setup.py

```diff
@@ -1,11 +1,11 @@
-## ! DO NOT MANUALLY INVOKE THIS setup.py, USE CATKIN INSTEAD
+# ! DO NOT MANUALLY INVOKE THIS setup.py, USE CATKIN INSTEAD
 
 from catkin_pkg.python_setup import generate_distutils_setup
-from setuptools import find_packages, setup
+from setuptools import setup
 
 d = generate_distutils_setup(
-    packages=find_packages('src'),
-    package_dir={'': 'src'}
+    packages=["moveit"],
+    package_dir={"": "python/src"}
```

Comment:
did you check if this still works correctly when doing catkin install? I thought you had to list all modules & submodules explicitly for it to work, hence the `find_packages`

Reply:
Maybe, you are right. I used this pattern successfully for MTC, but I'm not sure I tested with install space.
The more pressing issue is that we actually don't want to install a `moveit` package here, but only the sub-package `core` into the `moveit` package. I'm not yet sure how to do this...

Reply:
this is what I was talking about in the thread on the other PR. I think to do one-big moveit we'd have to move some of this stuff up a level in terms of directory structure. Maybe we can put it in the empty moveit (meta?)package? The setup.py as well as any top-level C++ bindings and pure python code living directly in the `moveit` namespace and not under any sub-namespace could live there. That's just an idea though I haven't looked into implementing that

Reply:
Ok I just updated things (check the PR) -- I think I have it working for the use cases we've considered, and I tested by making bindings for moveit_ros_occupancy_map_updater. They can be installed alongside moveit/core without any problems from what I can tell :crossed_fingers: